testdata/p4_16_samples/pna-dpdk-parser-state-err.p4

```diff
@@ -172,7 +172,7 @@ control MainControlImpl(
         hdr.ipv4.srcAddr = newAddr;
     }
     action add_on_miss_action2() {
-        add_entry(action_name="next_hop", action_params = {32w0, 32w1234}, expire_time_profile_id = user_meta.timeout);
+        add_entry(action_name="next_hop2", action_params = {32w0, 32w1234}, expire_time_profile_id = user_meta.timeout);
```

Comment:
Is this action_name change required for the issue 3636?

Reply:
No. It is coming from another for add_entry (PR 3614)